src/sentry/models/__init__.py

```diff
@@ -7,6 +7,7 @@
 from .apiscopes import *  # NOQA
 from .apitoken import *  # NOQA
 from .appconnectbuilds import *  # NOQA
+from .artifactbundle import *  # NOQA
```

Comment:
@asottile-sentry do we still need this? I really want to at least avoid new models to end up in here.

Reply:
I don't think so -- I think this part of the patch can be discarded